templates/states.yml

```
@@ -9,48 +9,48 @@
 states:
   - id: evaluated
     label: Evaluated
-    aliases: [evaluada]
+    aliases: [ 'evaluada', 'değerlendirildi', 'degerlendirildi' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)